v2/clients/http/interval.go

```diff
@@ -0,0 +1,82 @@
+//
+// Copyright (C) 2021 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package http
+
+import (
+	"context"
+	"net/url"
+	"path"
+	"strconv"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/errors"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2/clients/http/utils"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2/clients/interfaces"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2/dtos/common"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2/dtos/requests"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2/dtos/responses"
+)
+
+type IntervalClient struct {
+	baseUrl string
+}
+
+// NewIntervalClient creates an instance of IntervalClient
+func NewIntervalClient(baseUrl string) interfaces.IntervalClient {
+	return &IntervalClient{
+		baseUrl: baseUrl,
+	}
+}
+
+func (client IntervalClient) Add(ctx context.Context, reqs []requests.AddIntervalRequest) (
```

Comment:
Add GoDoc comments for public functions in this PR.

Reply:
Added GoDoc comments.